### PR TITLE
opencv3 compatibility

### PR DESCRIPTION
**Pull request by fmessmer**

### CODE REVIEW

File: cob_object_detection_visualizer/CMakeLists.txt [hunk 1]
```diff
@@ -132,7 +132,7 @@ include_directories(
 
 ## Declare a cpp executable
 add_executable(object_detection_visualizer ros/src/object_detection_visualizer_node.cpp)
-add_dependencies(object_detection_visualizer ${catkin_EXPORTED_TARGETS} ${PROJECT_NAME}_generate_messages_cpp)
+add_dependencies(object_detection_visualizer ${catkin_EXPORTED_TARGETS})
```
Comment (fmessmer) [inline cob_object_detection_visualizer/CMakeLists.txt:135]: nothing generated

File: cob_object_detection_visualizer/package.xml [hunk 2]
```diff
@@ -16,14 +16,11 @@
   <depend>cv_bridge</depend>
   <depend>eigen_conversions</depend>
   <depend>image_transport</depend>
-  <depend>libopencv-dev</depend>
```
Comment (fmessmer) [inline cob_object_detection_visualizer/package.xml:19]: see recommendation http://wiki.ros.org/opencv3#package.xml

File: cob_object_detection_visualizer/ros/src/object_detection_visualizer_node.cpp [hunk 3]
```diff
@@ -76,7 +76,7 @@
 
 // opencv
 #include <opencv/cv.h>
-#include <opencv/highgui.h>
+#include <opencv2/highgui.hpp>
```
Comment (fmessmer) [inline cob_object_detection_visualizer/ros/src/object_detection_visualizer_node.cpp]: see recommendation http://docs.opencv.org/master/db/dfa/tutorial_transition_guide.html#tutorial_transition_hints_headers

**Comment by fmessmer** [comment]
the travis-ci/push job fails because I accidentally created a branch in ipa320 but already removed it again...

**Review by ipa-rmb — APPROVED** [review]

**Pull request merged**